page-sobre-nosotros.php

```diff
@@ -0,0 +1,103 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <!-- main posts loop -->
+  <section id="posts">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+?>
+
+    <article <?php post_class('row margin-bottom-small'); ?> id="page-<?php the_ID(); ?>">
+
+      <div class="page-content col col-s-12 col-m-9 col-l-6">
+        <div class="margin-bottom-small">
+          <?php the_content(); ?>
+        </div>
+
+        <div class="margin-bottom-small">
+          <h3 class="margin-bottom-small"><?php echo __('[:es]Directorio[:en]Directory'); ?></h3>
+          <?php
+            if (qtranxf_getLanguage() === 'es') {
+              $about_directory = IGV_get_option('_igv_about_directory');
+            } else {
+              $about_directory = IGV_get_option('_igv_about_directory_en');
+            }
+            if (!empty($about_directory)) {
+              echo apply_filters('the_content', $about_directory);
+            }
+          ?>
+        </div>
+
+        <div class="margin-bottom-small">
+          <h3 class="margin-bottom-small"><?php echo __('[:es]Prensa[:en]Press'); ?></h3>
+<?php
+      $current_query = current_query();
+      if ($current_query->have_posts()) {
+        while ($current_query->have_posts()) {
+          $current_query->the_post();
+/*           $number = get_post_meta($post->ID, '_igv_number'); */
+          $program_files = get_post_meta($post->ID, '_igv_program_files');
+
+          if (!empty($program_files[0])) {
+            echo '<ul>';
+            foreach ($program_files[0] as $file) {
+              echo "<li><a href=" . $file['file'] . " target='_blank' class='font-underline'>File: " . __($file['text']) . "</a></li>";
+            }
```

Comment:
then why is it `File: text` like that instead of the number? or whatever they have in the pdf?